plenum/server/pool_req_handler.py

```diff
@@ -21,13 +22,22 @@ class PoolRequestHandler(LedgerRequestHandler):
     write_types = {NODE, }
 
```

Comment:
We also need checks that a Node can participate in BLS signing only if it has a valid PoP for its key.
We can not enable this right now, since nodes in live pools need to send PoP first.